Rakefile

```diff
@@ -0,0 +1,22 @@
+require 'data_mapper'
+require './app/data_mapper_setup'
```

Comment:
You need the models for this to work - either require them here or require the server instead